core/src/main/java/by/it/academy/homework/lesson2/task1/Solution.java

```diff
@@ -4,36 +4,35 @@
 import java.util.Scanner;
 
 public class Solution {
-	
-	static int pairs(int k, int[] arr) {
-		int pairsCount = 0;
-		for(int i = 0; i < arr.length; i++) {
-			for(int j = i + 1; j < arr.length; j++) {
-				if (Math.abs(arr[i] - arr[j]) == k) {
-					pairsCount++;
-				}
-			}
-		}
-		
-		return pairsCount;
-	}
 
-	private static final Scanner scanner = new Scanner(System.in);
+    static int pairs(int k, int[] arr) {
+        int pairsCount = 0;
+        for (int i = 0; i < arr.length; i++) {
+            for (int j = i + 1; j < arr.length; j++) {
+                if (Math.abs(arr[i] - arr[j]) == k) {
+                    pairsCount++;
+                }
+            }
+        }
+        return pairsCount;
+    }
 
-	public static void main(String[] args) throws IOException {
-		String[] nk = scanner.nextLine().split(" ");
-		int n = Integer.parseInt(nk[0]);
-		int k = Integer.parseInt(nk[1]);
-		int[] arr = new int[n];
-		String[] arrItems = scanner.nextLine().split(" ");
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
Initialize scanner in main, not as class field, or put scanner to the top of the class.